src/app/campo/campo.component.ts

```diff
@@ -7,23 +7,28 @@ import { DatosService } from '../datos.service'; /* esto escrito a mano */
   styleUrls: ['./campo.component.css']
 })
 export class CampoComponent {
- 
+
   pokemones: Array<Record<string, string | number>> = [];
 
   constructor(public DatosService: DatosService) { /* lo de los paréntes escrito a mano */
 
     var anteriorX: Array<number> = [];
     var anteriorY: Array<number> = [];
+    var duracionMovimiento: number = 500;
+    var duracionBucle: number = duracionMovimiento * (DatosService.numerosPokemonsVisibles + 1);
```

Comment:
Personalmente creo que es buena práctica añadir las unidades de medida en el nombre de las variables para no tener que deducirlo mientras lees el código

```suggestion
    var msDuracionMovimiento: number = 500;
    var msDuracionBucle: number = duracionMovimiento * (DatosService.numerosPokemonsVisibles + 1);
```

Reply:
Me gusta!